nicegui/elements/anywidget.js

```diff
@@ -0,0 +1,129 @@
+import { load_widget, load_css } from "widget";  // lib/anywidget/widget.js
+import { convertDynamicProperties } from "../../static/utils/dynamic_properties.js";
+
+export default {
+  template: "<div></div>",
+  mounted() {
+    this.init_widget();
+  },
+  methods: {
+    _log(...args) {
+      if (this._debug) {
+        console.log("NiceGUI-Anywidget", ...args);
+      }
+    },
+    init_widget() {
+      (async () => {
+        const emit_to_py = this.$emit;
+        const log = this._log;
+
+        // Implement AFM: https://anywidget.dev/en/afm/
+        // References:
+        // * Marimo AFM impl:
+        // https://github.com/marimo-team/marimo/blob/7f3023ff0caef22b2bf4c1b5a18ad1899bd40fa3/frontend/src/plugins/impl/anywidget/AnyWidgetPlugin.tsx#L161-L267
+        const model = {
+          attributes: { ...this.traits },
+          callbacks: {},
+          get: function (key) {
+            log('Getting value for', key, ':', this.attributes[key]);
+            const value = this.attributes[key];
```

Comment:
If this just `return value`, this raises an error:
```javascript
Uncaught (in promise) DataCloneError: Failed to execute 'structuredClone' on 'Window': #<Object> could not be cloned.
    at reembed (1b7ef5d7-1cf7-4947-8f06-0f33e5db9f95:29:20)
    at Object.render (1b7ef5d7-1cf7-4947-8f06-0f33e5db9f95:146:11)
    at anywidget.js:96:47
```

Reply:
We can use `cleanObject` from ../../static/utils/json.js for this purpose.